ontology.ttl

```diff
@@ -0,0 +1,163 @@
+                            rdfs:comment "Suggested property for this resource - this is not a strict requirement but may be used as an interface hint"@en .
+
+# USERS
+
+# EDPOP users are represented in the graph as the creators of collections and
+# annotations. Their representation in the graph functions as their "public
+# profile": this information can be displayed to show attribution.
+
+edpopcol:User a rdfs:Class ;
+              rdfs:subClassOf as:Person ;
+              rdfs:label "User"@en .
+
+# Properties that users can fill in
+edpopcol:User edpopcol:suggestedProperty    as:name ,
+                                            as:url ,
+                                            edpopcol:affiliation .
+
+as:Link edpopcol:suggestedProperty  as:name , as:href .
+
+
+# Affiliation (e.g. university, library...)
+edpopcol:affiliation a rdf:Property ;
+                     rdfs:label "affiliation"@en ;
+                     rdfs:domain edpopcol:User ;
+                     rdfs:range as:Organization ;
+                     rdfs:comment "An institution a person is affiliated with"@en .
+
+as:Organization edpopcol:suggestedProperty as:name , as:url .
+
+# APPLICATIONS
+
+as:Application edpopcol:suggestedProperty as:name , as:url .
+
+# ATTRIBUTION
+
+edpopcol:creator    rdfs:subPropertyOf dcterms:creator ;
+                    rdfs:range edpopcol:User ;
+                    rdfs:label "creator" ;
+                    rdfs:comment "User who made this contribution"@en .
+
+# COLLECTIONS
+
+edpopcol:Collection a rdfs:Class ;
+                    rdfs:subClassOf rdf:Bag ,
+                                    as:Collection ;
+                    rdfs:label "Collection"@en ;
+                    rdfs:comment "Collections are unordered sets of resources, assembled by users. They also function as a context in which annotations are made."@en .
+
+edpopcol:Collection edpopcol:suggestedProperty  as:name,
+                                                as:summary ,
+                                                edpopcol:creator .
+
+# ANNOTATIONS
+
+# Annotations are comments and corrections on resources.
+
+edpopcol:Annotation a rdfs:Class ;
+                    rdfs:subClassOf oa:Annotation ;
+                    rdfs:label "Annotation"@en ;
+                    rdfs:comment "Annotations are comments and corrections on resources"@en .
+
+edpopcol:Annotation edpopcol:suggestedProperty  oa:hasTarget ,
+                                                oa:hasBody ,
+                                                oa:motivatedBy ,
+                                                edpopcol:creator ,
+                                                as:generator ,
+                                                as:context ,
+                                                as:inReplyTo .
+
+# subset of oa:Motivation that is implemented in EDPOP.
+# May be expanded in the future.
+_:ImplementedMotivation owl:oneOf (
```

Comment:
Why is this a blank node?

Reply:
I only defined it to reduce the complexity of the statement that follows it (which restricts the motivations of annotations to this class). It doesn't need an IRI as far as I'm concerned, unless you disagree?

Reply:
Since it restricts the set of valid objects, I think it can be considered part of the public interface and it deserves some documentation. A proper IRI is not strictly necessary for that purpose, but likely to be a bit easier to reference to.